script.program.fritzact/default.py

```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+'''
+Documentation for the login procedure
+http://www.avm.de/de/Extern/files/session_id/AVM_Technical_Note_-_Session_ID.pdf
+
+Smart Home interface:
+https://avm.de/fileadmin/user_upload/Global/Service/Schnittstellen/AHA-HTTP-Interface.pdf
+'''
+
+import hashlib
+import os
+import requests
+import resources.lib.tools as t
+import resources.lib.slider as Slider
+
+import sys
+from time import time
+import urllib
+import xbmc
+import xbmcgui
+import xbmcaddon
+import xbmcplugin
+from xml.etree import ElementTree as ET
+import re
+
+__addon__ = xbmcaddon.Addon()
+__addonID__ = __addon__.getAddonInfo('id')
+__addonname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__path__ = __addon__.getAddonInfo('path')
```

Comment:
you could use `xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')).decode('utf-8')` here. That way you don't need to use `xbmc.translatePath` on all other locations that depend on __path__